mobile/src/modules/auth/store/actions.ts

```diff
@@ -43,6 +44,7 @@ export const createActiveAccount = createActionFn<ActiveAccountGeneratorData, Th
     }
     const address = getBurstAddressFromAccountId(id);
 
+    // @ts-ignore
```

Comment:
I think we should update Account class, because not all fields are required, most of them is optional

Reply:
Agreed, will do in a follow up PR.